lightning/exceptions.py

```diff
@@ -6,3 +6,13 @@ def __init__(self, message, errors):
         self.errors = errors
         # Display the errors
         print(errors)
+
+
+class CookieException(Exception):
+    def __init__(self, message, errors):
+        # Call Exception.__init__(message)
+        # to use the same Message header as the parent class
+        super().__init__(message)
+        self.errors = errors
+        # Display the errors
+        print(errors)
```

Comment:
Useless `print`.

Reply:
fixed in 8c543aa7094c15a53e877f4037bc152f2c1099b3